### DISCUSSION TITLE
Cannot load LARGE document from Rpubs.com in Google Chrome

### DISCUSSION THREAD

**Discussion by PhilSmith26** [quarto-dev/quarto-cli#4432] [Q&A]
I want to load this web page on Chrome: https://rpubs.com/PhilSmith26/1004208

It is a fairly big page. It will load without problems on Safari, but not on the new Chrome, though it did load on a previous Chrome version (I don't remember which one). I waited a long time for it to load, but to no avail. I have cleared cookies and the cache and rebooted, but it makes no difference. I have reset Chrome to its original settings and rebooted, but that also makes no difference. I have tried to load the page on Chrome on three different computers, but the page will not load on any of them. But as  I said it always loads fine on Safari.

As I said, this web page is big. I am able to load other pages from the same site, rpubs.com, pages that are smaller.  So is there a restriction in Chrome on the maximum page size? If so, how can I raise the maximum size? Or is there some other issue here? Have others had this problem?

Any help appreciated.

Philip

**Comment by mcanouil** [comment]
How does this relate to Quarto CLI?
It seems your issue is only about Chrome and big pages, thus you probably should ask for help elsewhere more focused on Chrome issues.
